testing/run-scripts/run.sh

```diff
@@ -17,7 +17,7 @@ while getopts n:t:d opt; do
       *)
           echo "$0 [-n NUM] [-t TASK] [-d]"
           echo "  -n: Instance number NUM.  Port numbers are offset by this much."
-          echo "  -t: Task named TASK.  One of: bash, copypaste"
+          echo "  -t: Task named TASK.  One of: bash, adventure"
```

Comment:
I think this file's a mistake of mine -- from a prior prototype.  The images are now named after the browser (`uproxy/chrome-dev`), instead of the task (`uproxy/bash`), as I think this script still assumes.

This isn't a TODO for you on this PR, just a note.